indra/literature/pubmed_client.py

```diff
@@ -307,12 +307,130 @@ def _find_elem_text(root, xpath_string):
     return None if elem is None else elem.text
 
 
-def _get_journal_info(medline_citation, get_issns_from_nlm):
+def _get_issue_info(journal: ET.Element):
+    # Issue info
+    issue = journal.find('JournalIssue')
+    issue_volume = _find_elem_text(issue, 'Volume')
+    issue_issue = _find_elem_text(issue, 'Issue')
+
+    issue_pub_date = issue.find('PubDate')
+    if issue_pub_date is not None:
+        # Get issue year
+        issue_year = _find_elem_text(issue_pub_date, 'Year')
+        issue_year = int(issue_year) if issue_year else None
+
+    else:
+        issue_year = None
+
+    return {
+        "issue_volume": issue_volume,
```

Comment:
the prefix of `issue_` is redundant since this dict is inside another dict with key `issue_dict` -> just make "volume", "issue", "year"